src/main/java/org/opensrp/web/config/security/filter/CrossSiteScriptingPreventionFilter.java

```diff
@@ -0,0 +1,84 @@
+package org.opensrp.web.config.security.filter;
+
+import com.google.gson.*;
+import org.owasp.encoder.Encode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Map;
+
+public class CrossSiteScriptingPreventionFilter implements Filter {
+
+	private static Logger logger = LoggerFactory.getLogger(CrossSiteScriptingPreventionFilter.class);
+
+	@Override
+	public void init(FilterConfig filterConfig) throws ServletException {
+
+	}
+
+	@Override
+	public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+			throws IOException, ServletException {
+		logger.debug("Inside CrossSiteScriptingPreventionFilter  ...............");
+		XssPreventionResponseWrapper xssPreventionResponseWrapper = new XssPreventionResponseWrapper(
+				(HttpServletResponse) response);
+		chain.doFilter(request, xssPreventionResponseWrapper);
+		try {
+			String responseString = xssPreventionResponseWrapper.getCaptureAsString();
+			JsonElement jsonElement = new JsonParser().parse(responseString);
```

Comment:
We are in the process of standardizing JSON libraries and GSON will be removed in the future. Could you migrate this to Jackson